test/tic_tac_toe_pair/handlers_test.clj

```diff
@@ -0,0 +1,81 @@
+(ns tic-tac-toe-pair.handlers-test
+    (:require [cheshire.core :as json]
+              [clojure.test :refer :all]
+              [ring.mock.request :as mock]
+              [tic-tac-toe-pair.api :refer [app]]))
+
+(deftest app-test
+  (testing "main route"
+    (let [response (app (mock/request :get "/"))]
+      (is (= 200 (:status response)))))
+  (testing "not-found route"
+    (let [response (app (mock/request :get "/invalid"))]
+      (is (= 404 (:status response))))))
+  
+(deftest player-endpoint-test
+  (testing "the /player endpoint"
+    (testing "when provided with a valid JSON body"
+      (let [player-info-map {:player-turn 1
+                             :player-1-mark :x}
+            player-info-json (json/encode player-info-map)
+            response (app (-> (mock/request :post "/player" player-info-json)
+                              (mock/content-type "application/json")))]
+        (testing "returns a 200 status"
+          (is (= 200 (:status response)))
+        (testing "returns a valid game map"
```

Comment:
It's a bit unexpected that the `/player` endpoint would return a game map as well. If you like the convenience of making one request to the backend to get a new game then consider changing your route to reflect that it returns more than just a player.

Reply:
Thanks Tom. I'll rename the endpoint to game instead.